### PR TITLE
Feature/tnation/asg threshold

### PR DESCRIPTION
**Pull request by tnation14**
# Overview

Adjust ASG thresholds so that Autoscaling does not as occur as a result of CPU/Memory usage. Also, stop forwarding unused ports from the Vagrant VM, and update the Route53 hosted zones (this is a deployed but uncommitted change from the DNS cutover in #10).

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -6,11 +6,6 @@ Vagrant.configure("2") do |config|
     # Ubuntu 14.04 LTS
     config.vm.box = "ubuntu/trusty64"
 
-    # Ports to the services
```
Comment (hectcastro) [inline Vagrantfile:9]: Was this change intended?

**Review by hectcastro — APPROVED** [review]

File: Vagrantfile (hunk 1 above)
Comment (tnation14) [reply Vagrantfile:9]: It is intended; once I removed `transit` and `chatta`, forwarding those ports became unnecessary.
